ash/src/entry.rs

```diff
@@ -255,16 +255,10 @@ impl vk::StaticFn {
 }
 
 #[derive(Clone, Debug)]
-pub struct MissingEntryPoint(&'static [u8]); // Slice without the NUL terminator
+pub struct MissingEntryPoint;
 impl std::fmt::Display for MissingEntryPoint {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::result::Result<(), std::fmt::Error> {
-        unsafe {
-            write!(
-                f,
-                "missing Vulkan entry point: {}",
-                std::str::from_utf8_unchecked(self.0)
-            )
-        }
+        write!(f, "missing Vulkan entry point")
```

Comment:
Perhaps be more specific here: ``"Cannot load `vkGetInstanceProcAddr` symbol from library"``?